cred-manager/src/sourcecred/service/sourcecred.service.ts

```diff
@@ -153,17 +155,29 @@ export class SourceCredService {
     contributionCalculationId: string,
     userCredDtos: UserCredDto[],
   ) {
+    // need to delete the user score and interval for the team before we insert new one
```

Comment:
From what I understood we are not doing that yet right?
Could you add a TODO on this comment or already add on this PR?

FYI, as we added delete cascade on the model, a simple way to delete previous user scores would simply be to delete all contribution calculations from a team before inserting a new one, this would already delete all related user scores and userintervals

Reply:
No its done, will commit as part of this PR